src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
```

Comment:
Copying parts of the previous comment from : https://github.com/KjellKod/g3log/pull/143/files

Since this might affect other users I will call on them to voice their opinion. They can diff these and voice any objections:
@jkhoogland , @cstamatopoulos , @ctapmex , @AndreasSchoenle
[original-look.txt](https://github.com/KjellKod/g3log/files/669377/original-look.txt)
[new-look.txt](https://github.com/KjellKod/g3log/files/669378/new-look.txt)